build.gradle.kts

```diff
@@ -50,7 +50,10 @@ dependencies {
     include("org.lanternpowered", "lmbda", "2.0.0") // used in events
 
     // Geyser dependency for the fun injector mixin :)))
-    modImplementation("org.geysermc.geyser:fabric:2.2.3-SNAPSHOT")
+    modCompileOnly("org.geysermc.geyser:fabric:2.2.3-SNAPSHOT") {
+        exclude(group = "io.netty")
+        exclude(group = "io.netty.incubator")
+    }
```

Comment:
would doing `modCompileOnly` on geyser-core and geyser-mod with `isTransitive = false` lead to less junk on the classpath? 

i recall seeing that #122 will be further improved with the merge, but doesn't hurt for now?

Reply:
sure, sounds good